### ISSUE TITLE
WeaponAccessory: "weight" and "structure_points" not available for some items

### ISSUE THREAD

**Issue by plu2k** [elyukai/optolith-database-schema#17]
"weight" and "structure_points" are mandatory fields for WeaponAccessory, but the values are not provided in the rules.

Example: The description Gürtelhalter (Aventurische Rüstkammer II, S. 121) doesn't provide values for either of these fields.

**Comment by elyukai** [comment]
Currently I‘d say that item *should* have a weight value. I‘ll double-check and ask Ulisses otherwise, if the missing of those values is intentional.

**Comment by plu2k** [comment]
OK, but I found no description for WeaponAccessories that seemingly just "missed" the complexity and weight values. The focus is completely on the rule detail:
![image](https://github.com/elyukai/optolith-database-schema/assets/34626587/d56c6ab5-47cf-4780-a427-797ca9455d2e)

**Comment by elyukai** [comment]
Yeah, I don‘t know. I‘ll probably just ask Ulisses. It‘s not the first time they got a longer list of questions from me. 🙃

**Comment by elyukai** [comment]
Antwort:

> es wurde nur nie festgelegt und ist daher Meisterentscheid
